cli/admin/src/main/java/io/pravega/cli/admin/controller/metadata/ControllerMetadataViewReaderInfoCommand.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Pravega Authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.pravega.cli.admin.controller.metadata;
+
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.utils.ZKHelper;
+
+public class ControllerMetadataViewReaderInfoCommand extends ControllerMetadataCommand {
+
+    /**
+     * Creates a new instance of the Command class.
+     *
+     * @param args The arguments for the command.
+     */
+    public ControllerMetadataViewReaderInfoCommand(CommandArgs args) {
+        super(args);
+    }
+
+    @Override
+    public void execute() throws Exception {
+        ensureArgCount(3);
+
+        final String hostId = getArg(0);
+        final String readerGroupName = getArg(1);
+        final String readerId = getArg(2);
```

Comment:
is there anything to be validated once we take these inputs?

Reply:
No need to validate the inputs. If inputs is wrong, then will not find zNode at given path and exception message will shows at cli.